### ISSUE TITLE
Run in CPO unclear

### ISSUE THREAD

**Issue by shriram** [brownplt/pyret-lang#1076]
Many teachers had no idea why their definitions weren't showing up in the REPL. The Run button is "out of frame" (they're focused on "below the bar") and its name is not meaningful, and the environment lacks discoverability.

I don't like the way Racket _pesters_ you if you have edits to the definitions since the last Run, but surely we can do better? Just a discrete place where it persistently says "out of date, click Run to load definitions", something like that?

I also saw a person assume they have to click Run instead of Enter in the REPL (because that's how you "load" a program…). I'm not sure much could be done there.

**Comment by schanzer** [comment]
I could imagine a solution where the button "raises" via a slight drop-shadow or something, thereby communicating a connection between the button and the definitions area. A more dramatic indicator would be to have a second, smaller line of text inside the Run button which communicates more information.

When there are new definitions, it says "Run (with new Definitions)". Otherwise, it says "Run (and clear Interactions)", or something to that effect.

**Comment by shriram** [comment]
I don't think we should have some really clever visual effect that itself then needs explanation. I think Racket's solution is the right one in principle. It's just more intrusive than I like. (I made the color recommendation for it, based on visual perception principles, and now I really regret that. It needs to be a bit more subtle so it doesn't drag your attention away when you are _choosing_ to ignore it.) It should also be easier to dismiss of.
